src/elexmodel/handlers/data/CombinedData.py

```diff
@@ -139,13 +139,20 @@ def get_nonreporting_units(self, percent_reporting_threshold, features_to_normal
             # fixed effects manually.
```

Comment:
I know this is an old comment, but can you make the language clearer in this line 137 note? You mean like "all units from one category of a fixed effect" or something right?  Maybe give example of all DC units with the fixed effect "postal code"

Reply:
done, let me know what you think about the updated comment.